sql/src/main/java/io/crate/protocols/postgres/types/BooleanType.java

```diff
@@ -35,7 +35,9 @@
 
     private static final int TYPE_LEN = 1;
     private static final int TYPE_MOD = -1;
-    private static final short DEFAULT_FORMAT_CODE = FormatCode.TEXT;
+
+    private final static byte[] TEXT_TRUE = new byte[] { 't' };
+    private final static byte[] TEXT_FALSE = new byte[] { 'f' };
     private static final Collection<ByteBuffer> TRUTH_VALUES = ImmutableSet.of(
```

Comment:
My point is that clients may send the value in any of these representations. For instance, the JDBC client sends boolean values as text '1'/'0'.